grouper/public_key.py

```diff
@@ -79,8 +89,55 @@ def delete_public_key(session, user_id, key_id):
         KeyNotFound if specified key wasn't found
```

Comment:
sorry, it would't let me common on the right line (81). just update to reflect that it removes the key as well as associated tags.